### DISCUSSION TITLE
Putty Software

### DISCUSSION THREAD

**Discussion by Anjumvj** [learndevopsonline/learndevopsonline#1545] [Q&A] [answered]
Hi Raghu,
I have created 3 instances for front end ,backend and DB. The problem is I cannot connect in putty using Ip address. The password "DevOps321"I am giving shows Access denied.Please find the attachment. 
![Screenshot 2023-10-29 112222](https://github.com/learndevopsonline/learndevopsonline/assets/130336724/6983d2f8-cdc5-4320-b221-e6a51f8b1516)

**Comment by sreedharm07** [comment]
its connected there is no problem
![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/ff32f961-c4a0-4af4-8c1b-b8b738e3363b)

**Reply by sreedharm07** [reply]
steps - 
1. open putty paste your public ip
2. enter username (centos)
3. enter password (DevOps321)
follow these steps

**Reply by Anjumvj** [reply]
When I gave DevOps321 as password it  shows denied. cannot proceed

**Reply by sreedharm07** [reply, marked as answer]
did you followed above steps which i mentioned ? 
please follow steps as it is because its logged in for me but not for you. that means you are not following proper steps, please try above steps as it is

**Reply by Anjumvj** [reply]
Now its connected. Thanks a lot
